egs/commonvoice/ASR/pruned_transducer_stateless7/decoder.py

```diff
@@ -0,0 +1,105 @@
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang)
```

Comment:
Could you replace it with a symlink?

Reply:
Also for other files like `joiner.py` and `model.py`.

Reply:
Sure.